tests/unit/utils.js

```diff
@@ -1,3 +1,31 @@
+import { isVue2 } from 'vue-demi';
+
+export const mount = async (component, { plugins, mixins, mocks, props }) => {
+  let wrapper;
+  if (isVue2) {
+    // eslint-disable-next-line no-shadow
+    const { createLocalVue, mount } = await import('@vue/test-utils');
+    const localVue = createLocalVue();
+    plugins.forEach((plugin) => {
+      localVue.use(...plugin);
+    });
+    wrapper = mount(component, {
+      localVue,
+      mixins,
+      mocks,
+      propsData: props,
+    });
+  } else {
+    // eslint-disable-next-line no-shadow
+    const { mount } = await import('@vue/test-utils');
```

Comment:
We probably do want to follow the no-shadow rule. You can rename imports like this:
```javascript
const { mount: originalMount } = await import('@vue/test-utils');
```